airbyte-integrations/connectors/source-k6-cloud/source_k6_cloud/k6_cloud.yaml

```diff
@@ -0,0 +1,99 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  organizations_selector:
+    extractor:
+      field_pointer: ["organizations"]
+  projects_selector:
+    extractor:
+      field_pointer: ["projects"]
+  tests_selector:
+    extractor:
+      field_pointer: ["k6-tests"]
```

Comment:
you can simplify here:
```yaml
  selector:
    extractor:
      field_pointer: 
        - "{{ options['name'] }}"
```

Then only rename the test stream.

Reply:
Thanks, done!